src/constants.ts

```diff
@@ -62,54 +63,65 @@ export const USERNAME_VALIDATOR_REGEX = new RegExp(/^[\w-]+$/);
 
 export const THREE_RARITIES_SHARE: RarityShare[] = [
   {
-    rarity: 'Common',
+    rarity: ComicRarity.Common,
     value: 70,
   },
   {
-    rarity: 'Rare',
+    rarity: ComicRarity.Rare,
     value: 25,
   },
   {
-    rarity: 'Legendary',
+    rarity: ComicRarity.Legendary,
     value: 5,
   },
 ];
 
 export const FIVE_RARITIES_SHARE: RarityShare[] = [
   {
-    rarity: 'Common',
+    rarity: ComicRarity.Common,
     value: 60,
   },
   {
-    rarity: 'Uncommon',
+    rarity: ComicRarity.Uncommon,
     value: 25,
   },
   {
-    rarity: 'Rare',
+    rarity: ComicRarity.Rare,
     value: 10,
   },
   {
-    rarity: 'Epic',
+    rarity: ComicRarity.Epic,
     value: 3,
   },
   {
-    rarity: 'Legendary',
+    rarity: ComicRarity.Legendary,
     value: 2,
   },
 ];
 
+export const ONE_RARITY_SHARE: RarityShare[] = [
+  {
+    rarity: ComicRarity.None,
+    value: 100,
+  },
+];
+
+export const MAX_SIGNATURES_PERCENT = 10;
+
 export const getRarityShareTable = (numberOfCovers: number) => {
   switch (numberOfCovers) {
     case 3:
       return THREE_RARITIES_SHARE;
     case 5:
       return FIVE_RARITIES_SHARE;
+    case 1:
+      return ONE_RARITY_SHARE;
     default:
       throw new Error('Unsupported number of rarities');
   }
 };
 
-export const getRarityShare = (numberOfCovers: number, rarity: ComicRarity) => {
+export const getRarityShare = (numberOfCovers: number, rarity: string) => {
   const shareTable = getRarityShareTable(numberOfCovers);
-  return shareTable.find((share) => share.rarity === rarity).value;
+  return shareTable.find((share) => share.rarity.toString() === rarity).value;
```

Comment:
Curious, why did we move away from an enum here?

Reply:
enum the instruction need is mapped to ``Int`` value where as prisma enum is mapped to ``string``, i will make change to contract later and then fix this stuff